presto-kinesis/src/main/java/io/prestosql/plugin/kinesis/KinesisColumnHandle.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kinesis;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.prestosql.decoder.DecoderColumnHandle;
+import io.prestosql.spi.connector.ColumnMetadata;
+import io.prestosql.spi.type.Type;
+
+import java.util.Objects;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Kinesis version of ConnectoColumnHandle. Keeps all the data of the columns of table formed from data received
+ * from kinesis.
+ */
+public class KinesisColumnHandle
+        implements DecoderColumnHandle, Comparable<KinesisColumnHandle>
+{
+    private String connectorId;
+    private final int ordinalPosition;
+
+    /**
+     * Column Name
+     */
+    private final String name;
+
+    /**
+     * Column type
+     */
+    private final Type type;
+
+    /**
+     * Mapping hint for the decoder. Can be null.
```

Comment:
Rather than adding a comment about nullability, annotate with `javax.annotation.Nullable`. Or better, replace it with `Optional`. We generally avoid using nullable fields.

Reply:
This implements DecoderColumnHandle interface, hence adding @Nullable notation